admin/src/main/java/com/exadel/frs/helpers/EmailSender.java

```diff
@@ -28,16 +28,12 @@
 @Service
 public class EmailSender {
 
-    @Value("${spring.mail.sender}")
-    private String frsEmail;
-
     @Autowired
     private JavaMailSender javaMailSender;
 
     public void sendMail(final String to, final String subject, final String message) {
         val msg = new SimpleMailMessage();
         msg.setTo(to);
-        msg.setFrom(frsEmail);
```

Comment:
Why did you decide to delete it? 
`From` field could contain different value from `username`, also `From` can contain Name of the sender, e.g. '"My Name" <ttt@gmail.com>'

Reply:
cos JavaMailSender bean get sender from property (application.yml) spring.mail.username automatically
![image](https://user-images.githubusercontent.com/64147071/98139865-bfd1d800-1ed5-11eb-98b5-9645c74e9ae5.png)